mycroft/configuration/mycroft.conf

```diff
@@ -125,7 +125,7 @@
       "user": "precise",
       "folder": "/home/precise/wakewords"
     },
-    "phoneme_duration": 120,
+    "phoneme_duration": 0.2,
```

Comment:
Comment this default.  0.2 what?  

What was the reason for changing units?  What happens if someone customized this setting?  Why not stick with milliseconds?